src/main/java/com/mraof/minestuck/effects/SoporSicknessEffect.java

```diff
@@ -0,0 +1,62 @@
+package com.mraof.minestuck.effects;
+
+import com.mraof.minestuck.Minestuck;
+import net.minecraft.world.effect.MobEffect;
+import net.minecraft.world.effect.MobEffectCategory;
+import net.minecraft.world.effect.MobEffectInstance;
+import net.minecraft.world.entity.LivingEntity;
+import net.minecraftforge.event.entity.living.MobEffectEvent;
+import net.minecraftforge.eventbus.api.Event;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+import net.minecraftforge.fml.common.Mod;
+
+@Mod.EventBusSubscriber(modid = Minestuck.MOD_ID, bus = Mod.EventBusSubscriber.Bus.FORGE)
+public class SoporSicknessEffect extends MobEffect
+{
+	protected SoporSicknessEffect()
+	{
+		super(MobEffectCategory.NEUTRAL, 0x47453d);
+	}
+	
+	@Override
+	public void applyEffectTick(LivingEntity target, int effectLevel)
+	{
+		super.applyEffectTick(target, effectLevel);
+		
+		damageTarget(target);
+	}
+	
+	public static void damageTarget(LivingEntity target)
+	{
+		if (target.getHealth() > 5.0F)
+		{
+			target.hurt(target.damageSources().generic(), 0.5F);
+		}
+	}
+	
+	public static boolean isntApplicableEffect(MobEffectInstance effectInstance, LivingEntity target)
+	{
+		MobEffectEvent.Applicable event = new MobEffectEvent.Applicable(target, effectInstance);
+		
+		return event.getResult() == Event.Result.DEFAULT;
```

Comment:
This doesn't look right. I think you actually want something else here.

Reply:
intellij had advised simplifying that statement, it compiled correctly so i didn't question it. open to alternative suggestion

Reply:
Could you explain what it is supposed to be doing?